src/components/card/Card.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { ksStyle } from '../../styles/basic/ksBasic';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1.0,
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: 'rgba(52, 52, 52, 0.8)',
```

Comment:
According to Figma (frame 14) background color should be transparent otherwise imo container should take 100% of screen width and 100% of screen height. 

In other words, component you made is not an overlay https://react-native-elements.github.io/react-native-elements/docs/overlay.html.

Reply:
Solved. The profile is displayed in a card with a transparent background. Previously, this was not possible due to the use of the drawer navigator, because this behavior is only available with the stack navigator.